functions/brinks/src/constants.js

```diff
@@ -0,0 +1,25 @@
+const constants = () => ({
```

Comment:
typically the convention for constants is uppercase ex: THAT.PRODUCT_TYPE.TICKET

Reply:
Yuk, the variable maybe, but yuk

Reply:
fixed in commit 7d382d0